src/mmds/src/token_headers.rs

```diff
@@ -139,5 +146,16 @@ mod tests {
                 "-60".to_string()
             ))
         );
+
+        // Uppercased headers
+        let mut map: HashMap<String, String> = HashMap::default();
+        map.insert(
+            TokenHeaders::X_METADATA_TOKEN_TTL_SECONDS
+                .to_string()
+                .to_uppercase(),
+            "60".to_string(),
+        );
```

Comment:
I think you meant:
```suggestion
        // Lowercased headers
        let mut map: HashMap<String, String> = HashMap::default();
        map.insert(
            TokenHeaders::X_METADATA_TOKEN_TTL_SECONDS
                .to_string()
                .to_lowercase(),
            "60".to_string(),
        );
```

Reply:
Nit: we could have this section just under the `valid headers` one, for improved readability.